control/metrics.go

```diff
@@ -91,16 +89,8 @@ func errorMetricDynamicElementHasNoName(value, ns string) error {
 	return fmt.Errorf("A dynamic element %s requires a name for namespace %s.", value, ns)
 }
 
-// listNotAllowedChars returns list of not allowed characters in metric's namespace as a string
-// which is used in construct errorMetricContainsNotAllowedChars as a recommendation
-// exemplary output: "brackets [( ) [ ] { }], spaces [ ], punctuations [. , ; ? !], slashes [| \ /], carets [^], quotations [" ` ']"
-func listNotAllowedChars() string {
-	var result string
-	for groupName, chars := range notAllowedChars {
-		result += fmt.Sprintf(" %s %s,", groupName, chars)
-	}
-	// trim the comma in the end
-	return strings.TrimSuffix(result, ",")
+func errorMetricElementHasTuple(value, ns string) error {
+	return fmt.Errorf("A element %s should not define tuple for namespace %s.", value, ns)
```

Comment:
This is to avoid misinterpretation of tuple - plugin should NOT expose metric with words in brackets separated by semicolon (e.g. '(sth;sth)`) which will be interpreted as a tuple in the GetMetrics(). 

So, basically restriction of not allowed characters are not longer needed, but the only case which needs to be checked in `validateNamespace` is that related with tuple. But, as you can see, probability of fulfillment [this condition in `containsTuple`](https://github.com/intelsdi-x/snap/pull/1220/files#diff-a1ff046a48fbfbca760e04470fc7ac73R561) by namespace's element is rather low.  See [tests](https://github.com/intelsdi-x/snap/pull/1220/files#diff-d03b6fc79a4cf76dee147f92b0cd76efR100) to find more how it work.